src/kem/kyber/serialize.rs

```diff
@@ -84,124 +80,188 @@ pub(super) fn deserialize_then_decompress_message(
     for (i, byte) in serialized.iter().enumerate() {
         for j in 0..8 {
             let coefficient_compressed = ((byte >> j) & 0x1) as KyberFieldElement;
-            re.coefficients[8 * i + j] = decompress_q::<1>(coefficient_compressed);
+            re.coefficients[8 * i + j] = decompress_q(1, coefficient_compressed);
         }
     }
 
     re
 }
 
 #[inline(always)]
-fn serialize_little_endian_4<const OUT_LEN: usize>(
+pub(super) fn serialize_uncompressed_ring_element(
```

Comment:
Can you add comments for these when changing the name. Per spec these are just instantiations of the serialize function.

Reply:
I'll do all the documentation in one go once the code is stable, I've opened https://github.com/cryspen/libcrux/issues/132 to track this.